src/scenes/Partners/Detail/PartnerDetail.tsx

```diff
@@ -60,21 +113,98 @@ export const PartnerDetail = () => {
                 <Skeleton width="75%" />
               )}
             </Typography>
-            {/* {partner ? (
+            <Tooltip title="Update Global Innovations Client">
               <IconButton
                 color="primary"
                 aria-label="edit partner"
-                onClick={editPartner}
+                onClick={() => editPartner('globalInnovationsClient')}
               >
                 <PencilCircledIcon />
               </IconButton>
-            ) : null} */}
+            </Tooltip>
+          </header>
+          <div className={classes.subheader}>
+            <Typography variant="h4">
+              {partner ? 'Partner Information' : <Skeleton width={200} />}
+            </Typography>
+            {partner && (
+              <Typography variant="body2" color="textSecondary">
+                Created {formatDate(partner.createdAt)}
+              </Typography>
+            )}
+          </div>
+          <Grid container spacing={1} alignItems="center">
+            <Grid item>
+              <DataButton
+                onClick={() => editPartner('pmcEntityCode')}
+                secured={partner?.pmcEntityCode}
+                redacted="You do not have permission to view PMC Entity Code"
+                children={`PMC Entity Code${
+                  partner?.pmcEntityCode.value
+                    ? `: ${partner.pmcEntityCode.value}`
+                    : ''
+                }`}
+              />
+            </Grid>
```

Comment:
Let's put status first, like the designs have it. Set the `empty` prop here too.

Reply:
Changed